target/linux/ipq40xx/image/mikrotik.mk

```diff
@@ -19,10 +19,21 @@ define Device/mikrotik_hap-ac2
 endef
 TARGET_DEVICES += mikrotik_hap-ac2
 
+define Device/mikrotik_ldf-5-ac
+	$(call Device/mikrotik_nor)
+	DEVICE_MODEL := LDF 5 ac
+	SOC := qcom-ipq4018
+	DEVICE_PACKAGES := ipq-wifi-mikrotik_ldf-5-ac -kmod-ath10k-ct \
+		kmod-ath10k-ct-smallbuffers rssileds -kmod-usb2 -kmod-usb3 \
+		-libusb-1.0
```

Comment:
Why?

Reply:
the unit has no USB connector. So no need for any usb support. When coming from a compilation for another device, with usb, this hopefully removes usb support. If not, please advise how to force usb to be absent when this device is chosen in menuconfig.

Reply:
Device related there is simply 'no usb'. But how to automatically force all usb related stuff 'off' in OpenWrt compilation is the question. Have experimented further and will remove the -libusb-1.0 because it is not capable of overruling a selection of libusb in menuconfig. So it has to be left to the developer to turn user space related usb stuff 'off'. There is no usb entry in 'Global Settings', like there is for y/n ipv6.